### ISSUE TITLE
[rocketmq-connect-hive] Build and Implement RocketMQ Connect Hive

### ISSUE THREAD

**Issue by aniketiq** [apache/rocketmq-externals#543]
@vongosling @duhenglucky @qqeasonchen

**Comment by duhenglucky** [comment]
@aniketiq Are you a student of GSoC? If so, please submit your proposal as soon as possible. Of course, if not, you are also welcome to participate in the development of the Hive connector.

**Comment by aniketiq** [comment]
@duhenglucky Yes! I'll be participating in the GSOC program. I've submitted the proposal to Google. Please let me know if I do need to send the proposal anywhere else.

**Comment by vongosling** [comment]
This(Issue) is a good start in your GSoC topic, I have reviewed and made some comments on your proposal.

**Comment by aniketiq** [comment]
> This(Issue) is a good start in your GSoC topic, I have reviewed and made some comments on your proposal.

@vongosling I have not received any comments on my shared Google Doc. It might be possible that the comments weren't saved properly.
